Action.c

```diff
@@ -213,26 +213,36 @@ static Htop_Reaction actionSortByTime(State* st) {
 
 static Htop_Reaction actionToggleKernelThreads(State* st) {
    st->settings->hideKernelThreads = !st->settings->hideKernelThreads;
+   st->settings->lastUpdate++;
```

Comment:
Is this necessary? What does change about the merged command line on toggle?

Reply:
The color rendering information for the threads: With kernel threads it's green, while without it's rendered as a normal thread.

Reply:
Yes. Strictly speaking, the update for user/kernel threads is not needed. I added it here regardless, to have the `lastUpdate` stamp consistently applied for batched updates (i.e. avoid having to think too much about if it is needed in any given place).